tools/external/wabt/src/tools/postpass.cc

```diff
@@ -152,7 +221,14 @@ int ProgramMain(int argc, char** argv) {
 
     if (Succeeded(result)) {
       size_t fixup = 0;
-      StripZeroedData(module, fixup);
+      auto pre_memory =  FillFromSegments(module.data_segments);
+      auto segments   = CreateSegments(pre_memory);
+      if (pre_memory != FillFromSegments(segments)) {
+        std::cerr << "Fractured Memory Failed, not applying optimizations" << std::endl;
```

Comment:
Is this an actual error? If not, no need to do `std::cerr`.

Reply:
True, it is a warning that your data segments are not quite right for the optimizations.  So, I wanted it to be `std::cerr` to not lose that if you were piping errors and warnings some where.

But the way changed how I was doing this has changed completely as there isn't a linker option that you would use per se to 'fix' the warning.  So, I'm not against just using `std::cout`.